introduction.md

```diff
@@ -0,0 +1 @@
+A true shinobi never introduces himself. His weapons are his introduction as well as his farewell.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.